plugin/pkg/scheduler/factory/plugins_test.go

```diff
@@ -39,3 +39,22 @@ func TestAlgorithmNameValidation(t *testing.T) {
 		}
 	}
 }
+
+func TestPriorityOverFlow(t *testing.T) {
+	var a, b int
+	a = 5000
```

Comment:
Makes sense. The only thought that I had was, this could be used anywhere, so did not name variables to reflect weight and total.